plugins/auth-backend/src/providers/aws-alb/provider.ts

```diff
@@ -150,9 +150,8 @@ export class AwsAlbAuthProvider implements AuthProviderRouteHandlers {
     }
 
     try {
-      const headers = getJWTHeaders(jwt);
-      const key = await this.getKey(headers.kid);
-      const claims = JWT.verify(jwt, key) as AwsAlbClaims;
+      const verifyResult = await jwtVerify(jwt, this.getKey);
```

Comment:
`getJWTHeaders` is unused now right? could remove it in that case

Reply:
Done